samples/extensions/full_screen_exclusive/full_screen_exclusive.h

```diff
@@ -0,0 +1,130 @@
+/* Copyright (c) 2023, Holochip Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Demonstrates and showcases full_screen_exclusive and related functionalities
+ */
+
+#pragma once
+
+#include "common/vk_common.h"
+#include "core/instance.h"
+#include "platform/application.h"
+
+class FullScreenExclusive : public vkb::Application
+{
+  private:
+	enum class SwapchainMode        // This enum class represents all for stages from full screen exclusive EXT selections
+	{
+		Default,
+		Windowed,
+		BorderlessFullscreen,
+		ExclusiveFullscreen
+	};
+
+	enum class ApplicationWindowMode        // This enum class represents the application window modes
+	{
+		Windowed,
+		Fullscreen
+	};
+
+	struct SwapchainDimensions
```

Comment:
Why do you introduce this struct?
You could just as well hold a `VkExtent2D` and a `VkFormat` in the Context struct.
At least the name of this struct is not appropriate, it holds more than just the dimensions.